node/engine/chainservice/eth_chainservice.go

```diff
@@ -135,39 +126,46 @@ func newEthChainService(chain ethChain, startBlock uint64, na *NitroAdjudicator.
 
 	// Prevent go routines from processing events before checkForMissedEvents completes
 	ecs.eventTracker.mu.Lock()
-
-	ecs.wg.Add(3)
-	go ecs.listenForEventLogs(errChan, eventSub, eventChan, eventQuery)
-	go ecs.listenForNewBlocks(errChan, newBlockSub, newBlockChan)
-	go ecs.listenForErrors(errChan)
-
-	// Search for any missed events emitted while this node was offline
-	err = ecs.checkForMissedEvents(startBlock)
-	if err != nil {
-		return nil, err
+	{
+		ecs.wg.Add(3)
+		go ecs.listenForEventLogs(errChan, eventSub, eventChan, eventQuery)
+		go ecs.listenForNewBlocks(errChan, newBlockSub, newBlockChan)
+		go ecs.listenForErrors(errChan)
+
+		// Search for any missed events emitted while this node was offline
+		err = ecs.checkForMissedEvents(startBlock)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
Do these braces actually do anything, or are they there just as a guide for the eye?

I removed them and things still seemed to work. I might suggest a function if you want to encapsulate this logic.

Reply:
I just used them to help make the code more readable and easier to understand what is enclosed by the `mutex.Lock()` and `mutex.Unlock()`. I will look at refactoring into a function. I was just hesitant to do that because its a one-time use function that will require 7 arguments

Reply:
I see. Not opposed to keeping them for readability. My editor automatically removed those braces when I was tinkering on a branch based on this :-/

Reply:
I'll just leave them removed as done by your PR